wcfsetup/install/files/acp/update_com.woltlab.wcf_5.3.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+use wcf\system\database\table\column\CharDatabaseTableColumn;
+use wcf\system\database\table\column\NotNullInt10DatabaseTableColumn;
+use wcf\system\database\table\DatabaseTable;
+use wcf\system\database\table\DatabaseTableChangeProcessor;
+use wcf\system\database\table\index\DatabaseTableIndex;
+use wcf\system\package\plugin\ScriptPackageInstallationPlugin;
+use wcf\system\WCF;
+
+/**
+ * Updates the database table layout from WoltLab Suite Core 5.2 to 5.3.
+ * 
+ * TODO: untested, potentially incomplete
+ * 
+ * @author	Florian Gail
+ * @copyright	2001-2020 WoltLab GmbH
+ * @license	GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @package	WoltLabSuite\Core
+ */
+
+$tables = [
+	DatabaseTable::create('wcf1_media')
+		->columns([
+			NotNullInt10DatabaseTableColumn::create('fileUpdateTime')
+				->defaultValue(0)
+		]),
+	DatabaseTable::create('wcf1_user')
+		->columns([
+			CharDatabaseTableColumn::create('emailConfirmed')
+				->length(40)
+		])
+		->indices([
+			DatabaseTableIndex::create('username')
+				->type(DatabaseTableIndex::UNIQUE_TYPE)
+				->columns(['username']),
+		]),
+];
+
+(new DatabaseTableChangeProcessor(
+/** @var ScriptPackageInstallationPlugin $this */
+	$this->installation->getPackage(),
+	$tables,
+	WCF::getDB()->getEditor())
+)->process();
+
+// Delete mail form during the upgrade to make it 100% non-functional.
```

Comment:
Should this delete the file from the file log? This probably should also be split into a dedicated update script.